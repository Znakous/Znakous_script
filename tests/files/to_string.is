aboba = 46
print(to_string(aboba))
print(to_string(4 + 5))
print(to_string(3 * 2 + 1) + "ffff")
print(parse_num(to_string(7*7) + "3") * 10)