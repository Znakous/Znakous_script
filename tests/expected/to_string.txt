4697ffff4930